apps/admin_audit/lib/Actions/Files.php

```diff
@@ -40,10 +40,10 @@ class Files extends Action {
 	 */
 	public function read(array $params): void {
 		$this->log(
-			'File accessed: "%s"',
+			'File accessed: "%s", id is: "%s"',
```

Comment:
Maybe move the ID before the path so it is more readable? I am thinking of situations where the path is long, so having the ID at the beginning of the log line would help quickly find it.

```suggestion
			'File with id "%s" accessed: "%s"',
```

Same logic for other logs?

Reply:
@artonge Sure ll move the id in front and path to the end